### PR TITLE
Feature branch name

### PR DESCRIPTION
**Pull request by sarcasticfringehead**
Temporary changes

### CODE REVIEW

File: test/spec/link_spec.rb [hunk 1]
```diff
@@ -55,4 +55,10 @@
     expect(a.target).to eq '_self'
   end
 
+  it 'should go to the location after being clicked' do
```
Comment (serialbandicoot) [inline test/spec/link_spec.rb]: This is a duplicate!

Comment (Mossington) [reply test/spec/link_spec.rb]: blaaaaa

Comment (sarcasticfringehead) [reply test/spec/link_spec.rb]: I know, I like it anyway

**Comment by serialbandicoot** [comment]
> Temporary changes

**Pull request closed without merging**